src/metrics.jl

```diff
@@ -28,6 +28,8 @@ struct CorrDist <: SemiMetric end
 
 struct ChiSqDist <: SemiMetric end
 struct KLDivergence <: PreMetric end
+struct KLDivergence <: PreMetric end
```

Comment:
Fixed in https://github.com/JuliaStats/Distances.jl/pull/74/files#diff-08d86a8ef0a417f2c2269c73f1dbfe5bL31